chainntnfs/btcdnotify/btcd.go

```diff
@@ -3,20 +3,24 @@ package btcdnotify
 import (
 	"errors"
 	"fmt"
+	"net"
```

Comment:
Since this commit is getting quite big, maybe an idea to split it into two? :)

Reply:
Not gonna review this file for now, as I reckon it's mostly the same changes as for `bitcoind`. We should really try to dedup some of the code, if possible.